aqt/installer.py

```diff
@@ -826,11 +827,18 @@ def run_installer(
     tasks = []
     for arc in archives:
         tasks.append((arc, base_dir, sevenzip, queue, keep))
+    original_sh = signal.signal(signal.SIGINT, signal.SIG_IGN)
     ctx = multiprocessing.get_context("spawn")
     pool = ctx.Pool(Settings.concurrency)
```

Comment:
Change signal behavior before calling `Pool` is problematic because we cannot stop  when stack.